misk/web/modules/config/src/core/containers/RootContainer.tsx

```diff
@@ -10,14 +11,40 @@ export namespace RootContainer {
 
 class RootContainer extends React.Component<RootContainer.Props> {
   public state = {
-    isOpen: false,
+    isOpen: false
   };
 
   private handleClick = () => {
     this.setState({ isOpen: !this.state.isOpen });
   };
 
+  componentDidMount() {
+    axios
+      .get("http://0.0.0.0:8080/_admin/config")
+      .then(response => {
+        // const newState = Object.assign({}, this.state, {
```

Comment:
commented out code : remove?

Reply:
The React implementation is half built in this PR and will be polished and code cleaned for the next one.